frontend/package.json

```diff
@@ -20,7 +20,7 @@
     "react-scripts": "5.0.1"
   },
   "scripts": {
-    "start": "env-cmd -f ../.env.development react-scripts start",
+    "start": "react-scripts start",
```

Comment:
so this is because now this option is on `docker-compose`, right?

Reply:
Absolutely! This was needed to load the `.env.development` file (as [dotenv](https://pypi.org/project/python-dotenv/) in python), but is now handled by `docker compose` described in the two `docker-compose-dev.yml`

Reply:
thanks :ok_hand: